Content.Client/LateJoin/LateJoinGui.cs

```diff
@@ -247,16 +247,20 @@ private void RebuildUI()
 
                         string? reason = null;
 
-                        if (value == 0 || !tracker.IsAllowed(prototype, out reason))
+                        if (tracker.IsAllowed(prototype, out reason) == false)
                         {
-                            jobButton.Disabled = true;
-
                             if (!string.IsNullOrEmpty(reason))
                             {
                                 jobButton.ToolTip = reason;
+                                jobButton.Disabled = true;
```

Comment:
This would make it so that if the player is not allowed to pick the job, but for some reason the `reason` message is empty, the player can still pick the job. You can just leave this line at the same location it was at first.